src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -2139,10 +2455,13 @@ JNI_METHOD(void, invoke)
     SuccessOrExit(err = commandSender->FinishCommand(convertedTimedRequestTimeoutMs != 0
                                                          ? Optional<uint16_t>(convertedTimedRequestTimeoutMs)
                                                          : Optional<uint16_t>::Missing()));
-    SuccessOrExit(err =
-                      commandSender->SendCommandRequest(device->GetSecureSession().Value(),
-                                                        imTimeoutMs != 0 ? MakeOptional(System::Clock::Milliseconds32(imTimeoutMs))
-                                                                         : Optional<System::Clock::Timeout>::Missing()));
+
+    SuccessOrExit(err = device->GetSecureSession().Value()->IsGroupSession()
+                      ? commandSender->SendGroupCommandRequest(device->GetSecureSession().Value())
```

Comment:
We need extra processing and check for concrete invoke path, we should have groupId, and we should have kGroupIdValid, please update invokeElement as well. thanks
       chip::app::CommandPathParams commandPath = { groupId, clusterId, commandId, (chip::app::CommandPathFlags::kGroupIdValid) };

Reply:
Oh.. I didn't check this part. thank you. I'll modify it.